core/templates/components/ck-editor-helpers/ck-editor-4-rte.component.ts

```diff
@@ -99,6 +119,16 @@ export class CkEditor4RteComponent implements AfterViewInit, OnChanges,
     // components may change without re-rendering each of the components,
     // in such cases, it is sufficient to update the ckeditor instance manually
     // with the latest value.
+    let value = this.value;
```

Comment:
Maybe just add a comment that will refer to the note at the top of this file. So that folks who just see this part understand it quicker. Ditto below.

Reply:
Done!